support-frontend/assets/pages/supporter-plus-landing/twoStepPages/firstStepLanding.tsx

```diff
@@ -30,14 +31,27 @@ import { AmountAndBenefits } from '../formSections/amountAndBenefits';
 import { LimitedPriceCards } from '../formSections/limitedPriceCards';
 import { SupporterPlusCheckoutScaffold } from './checkoutScaffold';
 
-const shorterBoxMargin = css`
+const boxShorterMargin = css`
 	:not(:last-child) {
 		${until.tablet} {
 			margin-bottom: ${space[2]}px;
 		}
 	}
 `;
 
+// TODO : re-factor SupporterPlusCheckoutScaffold so that we do not require negative margin here
```

Comment:
Might be worth describing what problem this is fixing in a comment.

Reply:
Ok done